### PR TITLE
OCPBUGS-36266: updating netobserv cli packet capture filter command

### PR DESCRIPTION
**Pull request by skrthomas**
Version(s):

4.12+
Issue:

https://issues.redhat.com/browse/OCPBUGS-36266
Link to docs preview:

Capturing packets: https://78209--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/network_observability/netobserv_cli/netobserv-cli-using#network-observability-cli-capturing-packets_netobserv-cli-using

Packet capture reference: https://78209--ocpdocs-pr.netlify.app/openshift-enterprise/latest/observability/network_observability/netobserv_cli/netobserv-cli-reference
QE review:
- [X] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by skrthomas** [comment]
@Amoghrd @jpinsonneau Can you PTAL at this update to the packet capture guidance and reference?

File: modules/network-observability-netobserv-cli-reference.adoc [hunk 1]
```diff
@@ -186,9 +186,9 @@ $ oc netobserv packets [<option>]
 .Packet capture filter option
 |===
 |Option| Description| Possible values| Mandatory| Default
-| `--filter`
-| Enable packet capture filtering.
-| `tcp`, `udp`, or `<port>` You can specify filtering options using a comma as delimeter. For example, `tcp,80` specifies the `tcp` protocol and port `80`.
+| `<protocol>`,`<port>`
+| Capture packets from a specific protocol and port pair. Use a comma as delimiter. For example, `tcp,80` specifies the `tcp` protocol and port `80`.
+| `tcp`, `udp`, and `<port>`
```
[the remark this answers is not in the document]
Comment (skrthomas) [reply modules/network-observability-netobserv-cli-reference.adoc]: I can definitely see that. Will shorten this table to remove L191

**Review by jpinsonneau — APPROVED** [review]
Looks good to me, thanks @skrthomas !

**Review by GroceryBoyJr — COMMENTED** [review]
LGTM

**Comment by bergerhoffer** [comment]
@skrthomas This should be CPd to 4.17 as well since it went into main, can you please CP it?

**Pull request merged**